x/auth/vesting/types/vesting_account_internal_test.go

```diff
@@ -0,0 +1,66 @@
+package types
```

Comment:
Why is this under `*_internal_test.go`? Can we just bundle it with `vesting_account_test.go`?

Reply:
It's because we also wanted to test private functions such as `computeClawback` directly